Makefile

```diff
@@ -1,8 +1,12 @@
 OPENFAAS_URL?=http://127.0.0.1:8080/
 
 swarm:
-	docker service rm stronghash env-test
+	docker service rm stronghash env-test test-secret
 kubernetes:
-	kubectl delete -n openfaas-fn deploy/stronghash ; kubectl delete -n openfaas-fn deploy/env-test
+	kubectl delete -n openfaas-fn deploy/stronghash ; kubectl delete -n openfaas-fn deploy/env-test; kubectl delete -n openfaas-fn deploy/test-secret
 test:
 	gateway_url=${OPENFAAS_URL} time go test -count=1 ./tests -v
+secrets:
+	echo "tDsdf7sFT45gs8D3gDGhg54" > ${HOME}/secrets/secret_api_test_key.txt
```

Comment:
Not sure we about the path `${HOME}/secrets` it won't exist on a lot of peoples computer. Maybe better to use `/tmp`